services/ResetDemo.php

```diff
@@ -0,0 +1,99 @@
+            $result = mysqli_query($conn, $createUserSql);
+            if (!$result){
+                error_log("Create user query failed: " . mysqli_error());
+            }
+
+            $grantPrivilegesSql = "GRANT ALL PRIVILEGES ON gradeplus.* TO 'gradeplusclient'@'localhost';";
+            $result = mysqli_query($conn, $grantPrivilegesSql);
+            if (!$result){
+                error_log("Grant privileges query failed: " . mysqli_error());
+            }
+            $result = mysqli_query($conn, "FLUSH PRIVILEGES");
+            if (!$result){
+                error_log("Flush privileges query failed: " . mysqli_error());
+            }
+        }
+
+        //Close admin connection
+        mysqli_close($conn);
+
+        // Create gradeplusclient connection
+        $conn = mysqli_connect('localhost', 'gradeplusclient', 'gradeplussql');
+        if (!$conn) {
+            error_log("Connection to MySQL as gradeplusclient failed: " . mysqli_connect_error());
+        }
+
+        // Create database if it does not exist
+        $createDbSql = "CREATE DATABASE IF NOT EXISTS gradeplus";
+        $result = mysqli_query($conn, $createDbSql);
+        mysqli_select_db($conn, 'gradeplus');
+        if (!$result){
+            error_log("Create database query failed: " . mysqli_error());
+        }
+
+        // Drop table if it exists
+        $resetTableSql = "DROP TABLE IF EXISTS login;";
+        $result = mysqli_query($conn, $resetTableSql);
+        if (!$result){
+            error_log("Drop table query failed: " . mysqli_error());
+        }
+
+        // Create table
+        $createTableSql = "
+        CREATE TABLE login (
+            username VARCHAR(50) PRIMARY KEY,
+            email VARCHAR(50),
+            password VARCHAR(50),
+            dname VARCHAR(50),
+            loggedin INT
+        );";
+        $result = mysqli_query($conn, $createTableSql);
+        if (!$result){
+            error_log("Create table query failed: " . mysqli_error());
+        }
+
+        // Insert dummy data
+        $insertDataSql = "
+        INSERT INTO login (username, email, password, dname, loggedin) VALUES
+        ('demo', 'demo@gradeplus.com', 'demo', 'Demo', 0),
+        ('admin', 'admin@gradeplus.com', 'admin', 'Administrator', 0);
+        ";
+        $result = mysqli_query($conn, $insertDataSql);
+        if (!$result){
+            error_log("Insert dummy data query failed: " . mysqli_error());
+        }
+
+        $success = 1;
+        $error = 0;
+    } catch (Exception $e) {
+        // SQL error
+        $success = 0;
+        $error = 1;
+    }
+
+    mysqli_close($conn);
+    header('Content-Type: application/json');
+    echo json_encode(["success" => $success,"error" => $error,"illegal" => 0]);
+} else {
+    // User is not authorized
+    header('Content-Type: application/json');
+    echo json_encode(["success" => 0,"error" => 0,"illegal" => 1]);
```

Comment:
Slight change of plans here, let us redirect to `illegal.php`
```suggestion
    header("Location: illegal.php");
```